src/tools/tidy/src/ui_tests.rs

```diff
@@ -100,23 +100,39 @@ fn check_entries(tests_path: &Path, bad: &mut bool) {
 }
 
 pub fn check(root_path: &Path, bless: bool, bad: &mut bool) {
+    let issues_txt_header = r#"============================================================
+    ⚠️⚠️⚠️NOTHING SHOULD EVER BE ADDED TO THIS LIST⚠️⚠️⚠️
+============================================================
+"#;
+
     let path = &root_path.join("tests");
     check_entries(&path, bad);
 
     // the list of files in ui tests that are allowed to start with `issue-XXXX`
     // BTreeSet because we would like a stable ordering so --bless works
-    let issues_list =
-        include!("issues.txt").map(|path| path.replace("/", std::path::MAIN_SEPARATOR_STR));
-    let issues: Vec<String> = Vec::from(issues_list.clone());
-    let is_sorted = issues.windows(2).all(|w| w[0] < w[1]);
+    let mut prev_line = String::new();
+    let mut is_sorted = true;
+    let allowed_issue_names: BTreeSet<_> = include_str!("issues.txt")
+        .lines()
+        .skip(issues_txt_header.lines().count()) // skip the header lines
+        .map(|line| {
+            let line = line.to_string();
+            if prev_line > line {
+                is_sorted = false;
+            }
+
+            prev_line = line.clone();
```

Comment:
FWIW, there's some needless copying going on here AFAICT. `BTreeSet<&'static str>` should be fine (and should be what you get from iterating lines).